docker/PhpDockerfile

```diff
@@ -33,7 +33,7 @@ RUN printf 'post_max_size = 500M\n' >> /usr/local/etc/php/conf.d/uploads.ini
 RUN printf 'memory_limit = 500M\n' >> /usr/local/etc/php/conf.d/uploads.ini
 
 # copy apache config
-COPY ./docker/vhost.conf /etc/apache2/sites-available/000-default.conf
+COPY ./vhost.conf /etc/apache2/sites-available/000-default.conf
```

Comment:
Similarly to the first comment, after PR #2 was merged this change must be reverted since context must remain in the base directory.